torch/cuda/sparse.py

```diff
@@ -0,0 +1,87 @@
+import torch
+from torch import _C
+from ..tensor import _TensorBase
+from torch.sparse import _SparseBase, _sparse_tensor_classes
+from . import _lazy_init, device
+
+
+if not hasattr(torch._C, 'CudaSparseDoubleTensorBase'):
+    # Define dummy base classes
+    for t in ['Double', 'Float', 'Long', 'Int', 'Short', 'Char', 'Byte', 'Half']:
+        tensor_name = 'CudaSparse{0}TensorBase'.format(t)
+        torch._C.__dict__[tensor_name] = type(tensor_name, (object,), {})
```

Comment:
good point. To be honest I just did the same as in `torch/cuda/__init__.py`. I'll add an `__init__` that raises an error there as well.